test/Sia.Gateway.Tests/Filters/EventFiltersTests.cs

```diff
@@ -51,7 +51,7 @@ public void Filter_WhenFilterIsEmpty_ReturnsInputQueryable()
             }.AsQueryable();
 
 
-            var result = serviceUnderTest.Filter(testInput);
+            var result = testInput.WithFilter(serviceUnderTest);
```

Comment:
On line 57 of this file is an assertion for the `WithFilter` outputs. 

```
Assert.AreEqual(2, result.Count());
```

Since we're only testing how a single `EventFilter` works against a single `Event` we don't need to worry about lists and can just test the true/false value of `EventFilter.IsMatchFor`